api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -186,6 +198,29 @@ message Bootstrap {
   // Specifies optional bootstrap extensions to be instantiated at startup time.
   // Each item contains extension specific configuration.
   repeated core.v3.TypedExtensionConfig bootstrap_extensions = 21;
+
+  // Configuration sources that will participate in
+  // *udpa.core.v1.ResourceLocator* authority resolution. The algorithm is as
+  // follows:
+  // 1. The authority field is taken from the *udpa.core.v1.ResourceLocator*, call
+  //    this *resource_authority*.
+  // 2. *resource_authority* is compared against the authorities in any peer
+  //    *ConfigSource*. The peer *ConfigSource* is the configuration source
+  //    message which would have been used unconditionally for resolution
+  //    with opaque resource names. If there is a match with an authority, the
+  //    peer *ConfigSource* message is used.
```

Comment:
OK I see, this answers my question above. I think in the final version this doc should be somewhere else and linked to from everywhere this is relevant? Otherwise this is pretty confusing?

Reply:
Yeah, I plan on doing a major documentation round before we consider `udpa://` ready for use.